source/onboard/mattermost-limits.rst

```diff
@@ -0,0 +1,46 @@
+Workspace limits
+================
+
+There may be limits applied to your Mattermost Cloud workspace, based on the type of subscription plan you have. These limits do not apply to self-hosted deployments.
+
+These limits are applied across all users in your team. We'll let you know when you're halfway from reaching a limit, as well as when you're closer to it. 
+
+When limits have been reached, data is archived. This data is not deleted, and can be accessed when you upgrade to a plan with higher limits. If you choose to downgrade your plan, for example from Mattermost Professional to Mattermost Starter, the limits will be applied to your existing data.
```

Comment:
Not a blocker to merge, but there may be a question that when you downgrade to Starter (either from trial or paid plan), what happens? Files/messages/cards is self-explanatory as you'll just see the most recent information. 

How about teams, integrations or views -- which ones will be retained upon downgrade?

Reply:
I'm following this up with John and it'll be in the next iteration of this content. :)